### PR TITLE
TRC-51 - chore:  Add `Mock Service Worker` and configure

### PR DESCRIPTION
**Pull request by mmcbride2929**
## TL/DR
Install, and configure `Mock Service Worker` so that the 'trending-suggestions' endpoint returns an array of  `MetaDataItem` objects.

## Overview of Changes

`package.json`
- install MSW

`.env`
- add NODE_ENV to declare development mode

`.gitignore`
- add to secure .env

`mockServiceWorker.js`
- initiated MSW with  `npx msw init public/ --save`

`TrendingSuggestionList.stories.tsx`
- removed args error from isFetched & isFetching.

`index.tsx`
- add conditional to enable `msw` during development mode

`src/mocks/browser.ts`
- configure worker

`/src/mocks/handlers.ts`
- configure request/response

### CODE REVIEW

File: apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.stories.tsx [hunk 1]
```diff
@@ -21,15 +21,11 @@ const Template: ComponentStory<typeof TrendingSuggestionList> = (args) => (
 export const Default = Template.bind({});
 Default.args = {
   isLoading: false,
-  isFetching: false,
-  isFetched: true,
```
Comment (wijohnst) [inline apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.stories.tsx:25]: Can you tell me more about why these were removed?

**Review by wijohnst — COMMENTED** [review]
Left a couple of questions. Good job!

File: apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.stories.tsx (hunk 1 above)
Comment (mmcbride2929) [reply apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.stories.tsx:25]: I removed these because in our last component `TrendingSuggestionList` we were passing `isFetched` and `isFetching` as props to our `TrendingItemSuggestion`, but I refactored it to pass isLoading right before pushing the last PR. 

After doing so I forgot to change the story, and it was causing an error which wouldn't let me continue with configuring MSW. I probably should have made the change its on PR.

File: apps/web/src/mocks/handlers.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { rest } from 'msw';
+import { metaDataItems } from '../components/TrendingSuggestionList/TrendingSuggestionList.utils';
+
+export const handlers = [
+  rest.get('/trending-suggestions', (req, res, ctx) => {
+    // how do we get prepped data?
```
[the remark this answers is not in the document]
Comment (mmcbride2929) [reply apps/web/src/mocks/handlers.ts:6]: This was a note to myself, that was also looked over and not deleted. I was wondering to myself, "where do we get the array of objects?", but remembered we had created a utils file.

**Review by wijohnst — APPROVED** [review]
🚀
